norduser/service/fork.go

```diff
@@ -199,3 +210,23 @@ func (c *ChildProcessNorduser) StopAll() {
 	case <-time.After(10 * time.Second):
 	}
 }
+
+func (c *ChildProcessNorduser) Restart(uid uint32) error {
```

Comment:
Have you checked how restart affects fileshare? Let's say that there's some big transfer in progress, user disables tray which restarts norduser/fileshare. My guess is that it shouldn't cause any issues since libdrop can resume transfers now, but it would be good to check.

Reply:
I haven't checked that.

Reply:
A new Jira ticket was created for that.